networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -18,7 +18,7 @@ Before you can use NMState with {product-title}, you must install the Kubernetes
 
 [NOTE]
 ====
-The Kubernetes NMState Operator updates the network configuration of a secondary NIC. It cannot update the network configuration of the primary NIC or the `br-ex` bridge.
+The Kubernetes NMState Operator updates the network configuration of a secondary NIC. The Operator cannot update the network configuration of the primary NIC, or update the `br-ex` bridge on most on-premise networks. On a bare-metal platform, using the Kubernetes NMState Operator to update the `br-ex` bridge network configuration is only supported if you set the br-ex` bridge as the interface in the NMState configuration in your `NodeNetworkConfigurationPolicy` manifest file for your cluster.
```

Comment:
It's not enough for it to be present in the NNCP. You need to have deployed initially with it in the machine-config (which unfortunately is separate from the NNCP).

Reply:
However, it is worth noting that if you make day 2 changes, you must be sure to include the br-ex configuration in your NNCP.

In fact, that day 2 process needs explicit documentation because it is also necessary to add the masquerade address to your bridge configuration on day 2. On day 1, OVNK will handle adding it, but if you make changes after OVNK is already running you need to make sure to configure it yourself.

This whole feature may warrant its own section of the docs because there is quite a bit we need to cover.

Reply:
:heavy_check_mark: